settings.gradle

```diff
@@ -14,7 +14,8 @@ include ":opentelemetry-all",
         ":opentelemetry-sdk",
         ":opentelemetry-sdk-contrib-async-processor",
         ":opentelemetry-sdk-contrib-aws-v1-support",
-        ":opentelemetry-sdk-contrib-testbed"
+        ":opentelemetry-sdk-contrib-testbed",
+        ":opentelemetry-sdk-contrib-auto-config"
```

Comment:
add it to "all" package as well.

Reply:
will do

Reply:
done, assuming I just needed to update build.gradle in the opentelemetry-all module